src/daemon/configfile.c

```diff
@@ -317,6 +317,38 @@ static int dispatch_loadplugin(oconfig_item_t *ci) {
   return ret_val;
 } /* int dispatch_value_loadplugin */
 
+static int dispatch_value_plugin(const char *plugin, oconfig_item_t *ci) {
```

Comment:
You're reintroducing this function, which was removed by d0e2590eec07925abdc656cadca205738e3471ed. Is this by accident?

Reply:
yes, totally by accident.